### ISSUE TITLE
Dropout: A simple way to prevent Neural Nets from overfitting

### ISSUE THREAD

**Issue by huangy22** [huangy22/ML-research-reading#3]
http://jmlr.org/papers/volume15/srivastava14a/srivastava14a.pdf
